### PR TITLE
Use ci setup script for MATLAB

### PR DESCRIPTION
**Pull request by jwpereira**

### CODE REVIEW

**Review by mcafaro — APPROVED** [review]
:shipit:

File: lib/travis/build/script/matlab.rb [hunk 1]
```diff
@@ -30,10 +30,14 @@ def setup
           end
 
           sh.fold 'matlab_install' do
+            sh.echo 'Installing MATLAB system dependencies', ansi: :yellow
```
Comment (mcafaro) [inline lib/travis/build/script/matlab.rb]: If this is what folks will see for the entire fold, does it make sense to just have 1 message that says something like "Installing MATLAB"? Then we could consider submessages like "Installing system dependencies", etc.

Comment (jwpereira) [reply lib/travis/build/script/matlab.rb]: We could do that 🤔

Comment (mcafaro) [reply lib/travis/build/script/matlab.rb]: What I'm thinking is that if we don't do something like that the fold will say "Installing MATLAB system dependencies" but under that fold it actually installs the system deps and MATLAB.

Comment (jwpereira) [reply lib/travis/build/script/matlab.rb]: That reasoning makes sense to me. I've updated the PR with that feedback 🤔

**Review by mcafaro — APPROVED** [review]

**Pull request closed without merging**